octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/field/FieldEditorFactory.java

```diff
@@ -248,7 +248,7 @@ private EntityLoader createGenericEntityLoader(Entity entity, EntityModelWrapper
                     sortingOrderByAsc = true;
             }
 
-            return entityService.findEntities(entity, qb, null, null, null, COMBO_BOX_ENTITY_LIMIT, sortingFields, sortingOrderByAsc);
+            return entityService.findEntities(entity, qb, null, null, null, COMBO_BOX_ENTITY_LIMIT, sortingFields, sortingOrderByAsc, null);

```

Comment:
why null?

Reply:
because I added a new param in the findEntities method (common module)